Python/binary-trees/binarytrees.py

```diff
@@ -73,4 +73,8 @@ def main(n, min_depth=4):
 
 
 if __name__ == '__main__':
```

Comment:
This is exactly what’s needed for the timing part but let’s keep it out for this patch.